clients/drcachesim/tracer/output.cpp

```diff
@@ -822,7 +819,9 @@ process_entry_for_physaddr(void *drcontext, per_thread_t *data, size_t header_si
         data->num_phys_markers +=
             output_buffer(drcontext, data, data->v2p_buf, v2p_ptr, header_size);
         v2p_ptr = data->v2p_buf;
-        v2p_ptr += add_buffer_header(drcontext, data, v2p_ptr);
```

Comment:
Remind me why we add a header the v2p buf too?

Reply:
IIRC, once we decided to make it a separate buffer, it is emitted before the regular one so we put a header on it.  With the two separate write-outs potentially not being atomic it seems best to have each buffer have its own header.